tcl/pdtk_canvas.tcl

```diff
@@ -117,7 +117,8 @@ proc pdtk_canvas_new {mytoplevel width height geometry editable} {
     canvas $tkcanvas -width $width -height $height \
         -highlightthickness 0 -scrollregion [list 0 0 $width $height] \
         -xscrollcommand "$mytoplevel.xscroll set" \
-        -yscrollcommand "$mytoplevel.yscroll set"
+        -yscrollcommand "$mytoplevel.yscroll set" \
+        -background white
```

Comment:
couldn't we just start using variables rather than hardcoding colours (in the spirit of #196)?